tsconfig.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
I like having this in place; let's rename it to `test.tsconfig.json` so that it doesn't get picked up on the editor side. We may want to actually migrate this repo to TS, so we need the test-specific config to be separate!